test/FunctionalTests/Microsoft.OData.Client.Tests/DataServiceQueryProviderTests.cs

```diff
@@ -218,11 +370,40 @@ private class Product
 
             public decimal Price { get; set; }
 
+            public Edm.Date LaunchDate { get; set; }
+
             public IEnumerable<string> Comments { get; set; }
 
             public static bool StaticFunction(string parameter) { return true; }
 
             public bool InstanceFunction(string parameter) { return true; }
+
+            [OriginalName("sale")]
+            [UriFunction]
+            public int YearSale(int year) => throw new NotSupportedException();
+
+            [UriFunction(true)]
+            public bool Test(string data)
+            {
+                return string.IsNullOrEmpty(data);
+            }
+        }
+
+        private static class UriFunctions
+        {
+            [OriginalName("now")]
+            [UriFunction]

```

Comment:
why cannot merge OriginalName and UriFunction one?

Reply:
Merged attributes on test classes as suggested.
Thanks.